### ISSUE TITLE
Can no longer connect to redditP with iPad

### ISSUE THREAD

**Issue by ExiledTriangle** [ubershmekel/redditp#131]
Love your site, use it every day. Two days ago I couldn't connect to redditP on my iPad2. Yesterday I couldn't either and started to troubleshoot - even though nothing had changed on the iPad before I lost the ability to connect to redditP. Changing DNS does nothing. Reset all network settings, did nothing. Tried a different WiFi router, did nothing. Safari says "Safari cannot open the page because it could not establish a secure connection to the server." Other browsers I have installed just sit there after typing in the address. In searching I discovered this url: https://utopiabound.github.io/redditp/?/ which is pretty cool (more options than I am used to), that page actually loads on the iPad but the slideshow does not - just says "loading" at the top & bottom. iOS version is 8.4 if that matters (been jailbroken from day 1 so I rarely update and only if I absolutely have to, usually to use a newer application) - but I think this is some sort of https/certificate error. redditP loads fine on my PCs and redditP is the only site I have this problem with on the iPad, everything else seems fine.

Any ideas what is going on? Did you or reddit change something recently?

**Comment by ubershmekel** [comment]
I wrote back in the reddit message. Maybe you need new root certificates installed. I can't say for sure. It does work on my ipad.

**Comment by ExiledTriangle** [comment]
I think June 9, 2021 was the last day I could connect. I discovered I can also not connect to https://www.netlify.com/

Exported and installed .cre and .crt files from both old.redditp.com and *.netlify.com but so far no luck. Right now I am focused on getting netlify.com to load hoping that solves the issue.

**Comment by ubershmekel** [comment]
Did you ever succeed in figuring this one out @ExiledTriangle ?

**Comment by ExiledTriangle** [comment]
Nope. Tried all kinds of things. After thinking about Apple's planned obsolescence, how I can no longer update my iPad2 past iOS 9 (circa 2016) - how with the internet in general certain browsers (like Legacy Firefox) don't even work with some sites now (Thanks Google! for your "Chrome Standard") - I decided it just wasn't worth the effort. Even if I took the time to upgrade from iOS 8.4 to 9.3.X another problem would likely popup in the future.

So I got a Amazon fireHD10, de-amazon'd it, installed Aurora Store so I can get stuff from Google Play without having to have a Google account (I hate Google and Apple), and now I am back to being able to use redditp on a tablet. When I signed into my Amazon account from Brave using the fireHD10, Amazon didn't even recognize the device as their own, it just said generic android device lol.

Funny you send this follow up now. I just replaced the digitizer on my iPad2 yesterday because over half of the screen was unresponsive - been sitting unused on my desk for weeks since I got the fireHD10. I can still use Kodi & some apps with it but forget about many internet sites. Wikipedia is another site I discovered it can't connect to anymore - there are others I don't recall right now, but the problem is more widespread than I originally realized.

I really think this was planned by Apple and the rest of Big Tech to force owners of these devices to have to upgrade. Technology simply isn't advancing as fast as it was in the 90's/00's. Most people don't need a new computer every year or so - but they will sure try to sell these NPCs one every year! Some will even camp in a line for days to get it. Such is life in Clown World.

Thanks for the follow-up and maintaining redditp!

**Comment by ubershmekel** [comment]
Glad to help. You can use the http version I have at http://gh.redditp.com/ and see if that works for you.
